.github/workflows/changelog-requirement.yml

```diff
@@ -25,11 +25,11 @@ jobs:
         with:
           files_yaml: |
             changelogs:
-              - 'changelog/unreleased/**/*.yml'
+              - 'changelog/unreleased/*/*.yml'
```

Comment:
In this context, "**" is a wildcard that matches zero or more directories in a file path, while "*" is a wildcard that matches any single directory or file name in a file path.

So, 'changelog/unreleased/**/*.yml' will match all YAML files in any subdirectory under changelog/unreleased, while 'changelog/unreleased/*.yml' will only match YAML files directly under changelog/unreleased.